File: randomNumber.cpp
#include <iostream>
#include <cstdlib>
#include <time.h>

using namespace std;
int main(){
	int n;
	cout << "Type a number" << endl;
	cin >> n;

	srand(time(NULL));
	int randomNumber = rand() % 10 + 1;
	for(int attemp = 10; attemp > 0; attemp--){
		if(n > randomNumber || n < randomNumber){
			cout << "You have " << attemp << " attemps, Try again" << endl;
			cin >> n;
		}
	}

	if(n == randomNumber){
		cout << "Good job! :D" << endl;
	} else {
		cout << "Game over" << endl;
		cout << "The random number is: " << randomNumber << endl;
	}
}

File: randomNumber.java
import java.util.Scanner;
import java.util.Random;
public class Main {
	public static void main(String[] args) {
		Scanner in = new Scanner(System.in);
		Random random = new Random();
		int randomNumber = random.nextInt(11);
		int n;
		
		System.out.println("Put a number");
		n = in.nextInt();
		
		for(int attemp = 10; attemp > 0; attemp--){
			if(n > randomNumber || n < randomNumber){
				System.out.println("You have " + attemp + " attemps, Try again");
				n = in.nextInt();
			}
		}
		
		if(n == randomNumber){
			System.out.println("Good job! :D");
		} else {
			System.out.println("Game over");
			System.out.println("The random number is: " + randomNumber);
		}
	}
}
